commit 0d3e7e1ca67146ec8b9ff37821484894b777b456
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Tue Nov 27 16:55:14 2018 +0000

    ARM: dts: bcm283x: Set downshift-after for Pi 3B+
    
    Enable the auto-downshift feature on Raspberry Pi 3B+ so that a link
    can eventually be established using a cable with pairs C and/or D
    missing or broken in a 1000Mbps-capable port.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/boot/dts/bcm283x-rpi-lan7515.dtsi b/arch/arm/boot/dts/bcm283x-rpi-lan7515.dtsi
index a925fa107a01..69d2c2757e45 100644
--- a/arch/arm/boot/dts/bcm283x-rpi-lan7515.dtsi
+++ b/arch/arm/boot/dts/bcm283x-rpi-lan7515.dtsi
@@ -31,6 +31,7 @@
 						reg = <1>;
 						microchip,eee-enabled;
 						microchip,tx-lpi-timer = <600>; /* non-aggressive*/
+						microchip,downshift-after = <2>;
 						microchip,led-modes = <
 							LAN78XX_LINK_1000_ACTIVITY
 							LAN78XX_LINK_10_100_ACTIVITY
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1323_0d3e7e1ca67146ec8b9ff37821484894b777b456.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1323_0d3e7e1ca67146ec8b9ff37821484894b777b456.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
